bundles/AdminBundle/Controller/Admin/TargetingController.php

```diff
@@ -240,7 +249,7 @@ public function targetGroupAddAction(Request $request, CoreCacheHandler $cache,
         $targetGroup->save();
 
         $event = new TargetGroupEvent($targetGroup);
-        $eventDispatcher->dispatch($event, TargetGroupEvents::POST_ADD);
+        $this->dispatchEvent($event, TargetGroupEvents::POST_ADD);
```

Comment:
This is a bit out of context, but we should also consider this in this PR. 
The `TargetGroupEvents` should be dispatched in the `save()` method of the model and not in the controller -> similar to other models. 
@kingjia90 can you change that accordingly? Thanks